packages/console/src/ConsoleInput.jsx

```diff
@@ -6,8 +6,9 @@ import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
 import Log from '@deephaven/log';
 import { PromiseUtils } from '@deephaven/utils';
 import { getCommandHistoryStorage } from '@deephaven/redux';
-import MonacoUtils from './monaco/MonacoUtils';
-import MonacoTheme from './monaco/MonacoTheme.module.scss';
+import MonacoUtils from '../monaco/MonacoUtils';
```

Comment:
Import shouldn't be changed

```suggestion
import MonacoUtils from './monaco/MonacoUtils';
```